### PR TITLE
Fixes for lint errors

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: module/components/Chip/index.js [hunk 1]
```diff
@@ -115,15 +115,14 @@ class Chip extends React.Component {
         }
 
         return (
-            <div className={classNames(className, defaultClasses)} onKeyDown={this.handleKeyDownForChip}
-                 onClick={this.onClick} {...attributes}>
+            <div className={classNames(className, defaultClasses)} onKeyDown={this.handleKeyDownForChip}onClick={this.onClick} {...attributes}>
```
Comment (dakshika) [inline module/components/Chip/index.js]: need space before onclick

**Review by dakshika — CHANGES_REQUESTED** [review]

File: module/components/Chip/index.js (hunk 1 above)
Comment (jeradrutnam) [inline module/components/Chip/index.js]: Add a space before `onClick=`

**Review by jeradrutnam — CHANGES_REQUESTED** [review]
Please do requested change

**Review by jeradrutnam — APPROVED** [review]
Approved

**Pull request merged**